script/utils/launch-docker.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+if (! docker stats --no-stream > /dev/null 2>&1  ); then
```

Comment:
Re this potentially not playing nice with Windows, I think it also won't play nice with Linux - maybe:

```
if [ "$(uname)" == "Darwin" ]; then
  if (! docker stats --no-stream > /dev/null 2>&1  ); then
    echo "==> Launching Docker..."

    # On Mac OS this would be the terminal command to launch Docker
    open /Applications/Docker.app
    printf "Waiting for Docker to launch..."
   #Wait until Docker daemon is running and has completed initialisation
  while (! docker stats --no-stream > /dev/null 2>&1 ); do
    # Docker takes a few seconds to initialize
    printf '.'
    sleep 1
  done
  fi
else
  echo "Not on Mac so skipping start Docker check"
fi
```